File: bench/loop.js
const start = Date.now();

let sum = 0;
for (let i = 1; i <= 1000; i++) {
  for (let j = 1; j <= 1000; j++) {
    sum += i * j;
  }
}

const elapsed = Date.now() - start;

console.log(`Loop sum (1000x1000) = ${sum} in ${elapsed}ms`);

File: bench/loop.rb
start = (Time.now.to_f * 1000).to_i

sum = 0
1.upto(1000) do |i|
  1.upto(1000) do |j|
    sum += i * j
  end
end

elapsed = (Time.now.to_f * 1000).to_i - start

puts "Loop sum (1000x1000) = #{sum} in #{elapsed}ms"
